### DISCUSSION TITLE
Are there git tags that correspond to the expected code for a publicly released client app or what is running in prod?

### DISCUSSION THREAD

**Discussion by feralbob** [cds-snc/covid-alert-documentation#54] [Q&A] [answered]
Hi All,

Given the challenges or maintaining protobuf's for V1/V2 I have a few questions. There appear to be 2 tags v1.0.0 and v.1.0.1. The latter occurred on Fri 28th Aug 2020.

- Are there are any tags for the server that correspond to what the released iOS client expects? 
- Or what is currently running in prod?
- Is HEAD of master expected to work correctly with HEAD of master on the client?
- Is there anyway for the public to verify what code is actually in prod?

Thanks for any info.

**Comment by obrien-j** [comment]
Good question @feralbob , I believe so, but will confirm with @CalvinRodo and @timarney as to how they're managing that aspect of the relEng process.

**Reply by CalvinRodo** [reply]
I wish I had a good answer to this, at the moment the only place that you can see the version that is released into production is in a private manifest repository. 

I will add that at the head of master in this repo is always releasable and so should always work with any client. 

I'll add on my backlog the need for a public repo that specifies the version that's running in production. 

I've also had on my backlog for a while to come up with a better release engineering process, as right now it's not as transparent or dev-friendly as I would like. 

I've been working on a way to automatically generate tags on master based on the conventional commit specification but as happens this has unfortunately slipped down the list of priorities for the server until I get a bit of breathing room. 

For reference and transparency at the moment: 

https://github.com/cds-snc/covid-alert-server/commit/7f9cfe159cb5797effdf89c6526713e42ad66096

Is released in production, this also happens to be the current head of the repo.

**Reply by feralbob** [reply]
Thanks @CalvinRodo and @obrien-j

**Reply by CalvinRodo** [reply]
Hey just realized i totally forgot that we actually report the version of the server running through an endpoint you can check that here: 

https://retrieval.covid-notification.alpha.canada.ca/services/version.json
https://submission.covid-notification.alpha.canada.ca/services/version.json

**Comment by feralbob** [comment, marked as answer]
Quoting @CalvinRodo in order for me to mark this as as the answer

> Hey just realized i totally forgot that we actually report the version of the server running through an endpoint you can check that here:
> 
> https://retrieval.covid-notification.alpha.canada.ca/services/version.json
> https://submission.covid-notification.alpha.canada.ca/services/version.json
